src/UIComponents/SaveModel.jsx

```diff
@@ -51,19 +50,31 @@ class SaveModel extends Component {
       });
     }
     return fields;
-  }
+  };
 
   getUsesFields = () => {
     var fields = [];
     fields.push({
       id: "potentialUses",
-      text: "How can this model be used?",
+      text: "Intended Use",
+      description:
+        "Describe the problem you think this model could help solve, or one potential app someone could make with this model.",
       placeholder: "Write a brief description."
     });
     fields.push({
       id: "potentialMisuses",
-      text: "How can this model be potentially misused?",
-      description: "Consider whether this model was trained on data that can identify subgroups, whether the data has adequate representation of subgroups, and whether this data could be used to inform decisions central to human life.",
+      text: "Warnings",
+      description: (
+        <div>
+          Describe any situations where this model could potentially be misused,
+          or any places where bias could potentially show up in the model.
+          Important questions to consider are:
+          <ul style={{ marginBottom: 0 }}>
+            <li>Is there enough data to create an accurate model?</li>
+            <li>Does the data represent all possible users and scenarios?</li>
+          </ul>
+        </div>
```

Comment:
I know we're not localising just yet, but this will be particularly difficult since I don't think we want to put JSX through the pipeline.  I wonder if it's worth already breaking this up into a few separate strings with no formatting involved...

Reply:
Seconded. I think we also want to avoid inline styling, per https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react

Reply:
Thank you both! I reviewed the style guide and implemented this feedback.